DTS-STN/passport-status
# Middleware to add /en to URL

## [ADO-685](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/685)

### Description

I've added a middleware that intercepts URL requests with the `default` locale (a dummy locale that was created since Next ignores adding the `defaultLocale` to the URL) and appends `/en` in front of the path.

This is the current official way of doing this in Next, which you can read more about [here](https://nextjs.org/docs/advanced-features/i18n-routing#prefixing-the-default-locale).

I've also updated the metadata title to what Steph mentioned. Since the `Index` page is supposed to be bilingual, I added a condition to the middleware that intercepts requests to `/en` or `/fr` and sends the user back to `/`. I also changed the metadata to be bilingual.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Upon starting the app you will be brought to `http://localhost:3000/`, ensure that when you add `en` or `fr` to the end of that links that you are redirected back to `/`
4. Click `English` or `Francais`, you should see the proper language in the path for both the landing and status pages

### Additional Notes
